### ISSUE TITLE
feature request : support for lme4 models

### ISSUE THREAD

**Issue by aghaynes** [cardiomoon/ggiraphExtra#7]
It would be great if e.g. ggPredict worked on lmer models… any plans to do that? It currently just gives an error : `Error: $ operator not defined for this S4 class`

**Comment by joomango** [comment]
I agree! It will be great to incorporate some linear mixed effect model for visualization with this package.
